test/demo_db/demo_db_test.cpp

```diff
@@ -97,15 +97,6 @@ TEST_F(DemoDBTest, DeleteRelTest) {
     ASSERT_EQ(TestHelper::convertResultToString(*result), groundTruth);
 }
 
-TEST_F(DemoDBTest, DeleteWithExceptionTest) {
```

Comment:
Why is this removed?

Reply:
I'm not sure, that was something @acquamarin did in [this commit](https://github.com/kuzudb/kuzu/pull/1573/commits/f35d4af43dff6420a254183777bbd628cc3c04da). @acquamarin?